doc/guides/feature-request-management.md

```diff
@@ -0,0 +1,89 @@
+# Feature request management
+
+Feature requests are a valuable source of input to the project.
+They help our maintainers understand what additions will be of
+value to users of the Node.js runtime.
+
+At the same time, the project is volunteer run and does not
+have the ability to `direct` resource at specific work. The
+features which are implemented are those for which volunteers
+are individually motivated to work on. The best way to ensure
+a feature gets implemented is to create a PR to add it.
+The project stives to support people who do that.
+
+An open feature request does not provide any indication that work
+on that feature will take place and after some period of time
+may be detrimental as it may result in an expectation that will
+never be fullfilled.
+
+This process tries to balance retaining the valuable input
+we get through feature requests and the overhead of
+maintaining open features requests that will never get
+implemented.
+
+## Creating feature requests
+
+A feature request can be created by adding the `feature request`
+lable to an issue. This may be done when the issue is opened or
+at a later point (often when investigation of a bug report
+results in it being considered a feature request as opposed to a
+bug).
```

Comment:
```suggestion
A feature request can be created by adding the `feature request`
label to an issue. This may be done automatically when the issue 
is opened or at a later point (often when investigation of a bug
report results in it being considered a feature request as opposed
to a bug).
```